docs/content/guides/developer/advanced/custom-indexer.mdx

```diff
@@ -37,6 +37,8 @@ The most straightforward stream source is to subscribe to a remote store of chec
 - Testnet: `https://checkpoints.testnet.sui.io`
 - Mainnet: `https://checkpoints.mainnet.sui.io`
 
+The checkpoint files are stored in the following format: `https://checkpoints.testnet.sui.io/<checkpoint_id>.chk`. You can download the checkpoint file by sending an HTTP GET request to the URL. [Try it yourself](https://checkpoints.testnet.sui.io/1.chk).
```

Comment:
```suggestion
The checkpoint files are stored in the following format: `https://checkpoints.testnet.sui.io/<checkpoint_id>.chk`. You can download the checkpoint file by sending an HTTP GET request to the relevant URL. Try it yourself for checkpoint 1 at [https://checkpoints.testnet.sui.io/1.chk](https://checkpoints.testnet.sui.io/1.chk).
```
nit. I think it'd be helpful to show the url in this case instead of hiding behind link text. Not sure my suggested wording is best, tho...